stack/services/src/main/java/org/apache/usergrid/services/notifications/QueueListener.java

```diff
@@ -156,7 +156,7 @@ private void execute(){
             try {
 
                 Timer.Context timerContext = timer.time();
-                List<QueueMessage> messages = queueManager.getMessages(getBatchSize(), MESSAGE_TRANSACTION_TIMEOUT, 5000, ApplicationQueueMessage.class);
+                List<QueueMessage> messages = queueManager.getMessages(getBatchSize(), MESSAGE_TRANSACTION_TIMEOUT, 10000, ApplicationQueueMessage.class);
```

Comment:
Yeah, I agree.  Given that this is something that is constantly producing a stream of output, we should probably change our queue manager to return an observable for subscription that will internally wrap the list implementation.